app/frontend/components/pages/dashboard/dashboardPage.tsx

```diff
@@ -160,11 +162,20 @@ const AccountsPage = ({screenType}: {screenType: ScreenType}) => {
   )
 }
 
+const MobileSendAdaPage = () => (
+  <Fragment>
+    <SendAdaPage />
+    <div className="mobile-multi-assets-page-wrapper">
```

Comment:
Are there some conventions for naming classes?

Reply:
I'm not aware of any.